### PR TITLE
onefetch: update to 2.18.1

### PR DESCRIPTION
**Pull request by cinerea0**
#### Testing the changes
- I tested the changes in this PR: **YES**

### CODE REVIEW

**Comment by icp1994** [comment]
There is a small bug https://github.com/o2sh/onefetch/issues/1092 - maybe a patch release soon.

**Comment by cinerea0** [comment]
Marking as draft until a patch release (or even just a commit with a fix) is released.

File: srcpkgs/onefetch/template [hunk 1]
```diff
@@ -12,7 +12,7 @@ license="MIT"
 homepage="https://github.com/o2sh/onefetch"
```
Comment (icp1994) [inline srcpkgs/onefetch/template:12]:
```suggestion
homepage="https://onefetch.dev"
```

File: srcpkgs/onefetch/template [hunk 2]
```diff
@@ -12,7 +12,7 @@ license="MIT"
...
 changelog="https://github.com/o2sh/onefetch/raw/main/CHANGELOG.md"
 distfiles="https://github.com/o2sh/onefetch/archive/${version}.tar.gz"
```
Comment (icp1994) [inline srcpkgs/onefetch/template:14]:
```suggestion
distfiles="https://github.com/o2sh/onefetch/archive/refs/tags/${version}.tar.gz"
```

**Review by icp1994 — COMMENTED** [review]
Fixed with release 2.18.1

**Pull request merged**